packages/models/src/Callback.ts

```diff
@@ -13,4 +13,13 @@ export interface Callback {
   isLogicFunctionProvided: boolean
   renderArguments: string[]
   isRenderFunctionProvided: boolean
+  results: CallbackResult[]
 }
+
+export type EntityValue = string | number | boolean | object
+
+export interface CallbackResult {
+  name: string
+  entityValues: Record<string, EntityValue | EntityValue[] | null | undefined>
```

Comment:
Do you need `string` in the type list since `EntityValue` can take a `string` ?

Reply:
I'm not sure what you mean, `string` is included in `EntityValue`

Reply:
Yep, see that.  Still had a foggy head yesterday.